ggml/src/CMakeLists.txt

```diff
@@ -566,6 +566,17 @@ if (GGML_SYCL)
         list(APPEND GGML_EXTRA_LIBS_PRIVATE DNNL::dnnl)
     endif()
 
+    set(oneCCL_DIR "/opt/intel/oneapi/ccl/latest/lib/cmake/oneCCL")
+    set(MPI_INCLUDE_PATH "/opt/intel/oneapi/mpi/latest/include")
+    set(MPI_LIBRARY_PATH "/opt/intel/oneapi/mpi/latest/lib/")
+    set(ONECCL_INCLUDE_PATH "/opt/intel/oneapi/ccl/latest/include")
+    set(ONECCL_LIBRARY_PATH "/opt/intel/oneapi/ccl/latest/lib/")
+    include_directories(${MPI_INCLUDE_PATH} ${ONECCL_INCLUDE_PATH})
+    find_library(MPI_LIBRARY mpi HINTS ${MPI_LIBRARY_PATH})
+    find_library(ONECCL_LIBRARY ccl HINTS ${ONECCL_LIBRARY_PATH})
+    # find_package(oneCCL REQUIRED)
+    message("-- oneCCL found")
+    set(GGML_EXTRA_LIBS ${GGML_EXTRA_LIBS} ${MPI_LIBRARY_PATH} ${ONECCL_LIBRARY_PATH})
```

Comment:
GGML_EXTRA_LIBS was recently split into GGML_EXTRA_LIBS_PUBLIC and GGML_EXTRA_LIBS_PRIVATE, so I think the line above won't work anymore
Also why there are paths to the lib directories inside this variable instead of found mpi/ccl libraries?